pom.xml

```diff
@@ -51,6 +51,17 @@
             <artifactId>github</artifactId>
             <version>1.14.0</version>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>branch-api</artifactId>
+            <version>0.2-beta-6</version>
```

Comment:
@jglick well, using this ItemListener approach, we need some way to get the SCMNavigators, if you think it best to add another interface for it, that's fine. Why should it not depend on the branch-api? It's providing directly related implementations...